registration/lib/auth.tsx

```diff
@@ -16,8 +16,8 @@ export const login = async ({ email, password }, expires = 1) => {
         `,
         variables: { email, password }
     })
-    debugger
-    if (graphqlMutation.errors.length) {
+
+    if (graphqlMutation.errors && graphqlMutation.errors.length) {
```

Comment:
Also, graphqlMutation may be null you need to check for it, you can try conditional chaining like `graphqlMutation?.errors?.length`. [It's new](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
`Add @babel/plugin-proposal-optional-chaining (https://git.io/vb4Sk) to the 'plugins' section of your Babel config to enable transformation.` I'll skip this one for now.